### DISCUSSION TITLE
option to let quickfix add import statements in top cell

### DISCUSSION THREAD

**Discussion by yuuuxt** [microsoft/vscode-jupyter#11313] [New Ideas]
Currently the quickfix only adds import statement on top of the current cell. 

Is there an option that can let quickfix always add import statements to the top cell?

**Comment by IanMatthewHuff** [comment]
@yuuuxt This could be an option to consider. If you don't mind I'm going to transfer this discussion into an issue in our repo. It will be easier for us to track it as a feature request versus just as a discussion here.

**Reply by yuuuxt** [reply]
Sure
